src/PhpSpreadsheet/Calculation/LookupRef.php

```diff
@@ -719,7 +719,6 @@ public static function VLOOKUP($lookup_value, $lookup_array, $index_number, $not
                 (!is_numeric($lookup_value) && !is_numeric($rowData[$firstColumn]))) {
                 if ($not_exact_match) {
                     $rowNumber = $rowKey;
-                    $rowValue = $rowData[$firstColumn];
 
                     continue;
                 } elseif ((strtolower($rowData[$firstColumn]) == strtolower($lookup_value))
```

Comment:
Not related to the bug that you're fixing, but it's probably worth changing these from `strtolower()` to use the `StringHelper::strToLower()` to make them multibyte-safe

Reply:
Will try to do that over the week-end.